Docs/todo.swagger.yml

```
@@ -0,0 +1,130 @@
+openapi: 3.1.0
+
+info:
+  title: Task API
+  description: Todoアプリでタスクの一覧表示、詳細表示、作成、編集、削除ができる
+  version: 1.0.0
+
+severs:
+  - url: http:localhost:3000/api/v1
+    description: Development sever
+
+tags:
+  name: task
```

書き方は下記にしてください。

``````suggestion
- name: task
``````